tests/specs/functional/infinite-gas-spec.k

```diff
@@ -1,15 +1,15 @@
 requires "infinite-gas.md"
-requires "lemmas/int-simplification.k"
+requires "lemmas/lemmas.k"
```

Comment:
This hierarchy seemed more intuitive to me. Is there any drawback to bringing `lemmas.k` in instead of just `int-simplification.k`?

Reply:
It just becomes a less focused testing harness, because `lemmas.k` introduces a bunch of extra lemmas about other thingns.

Reply:
So keep it like this?

Reply:
If we can switch it back, I would prefer that. If it only passes with this change, then we can keep it.

Reply:
A lot of the lemmas that were in `infitinite-gas.md` I moved to lemmas.k`, because they were referring to finite gas. But I could move them to `gas.md` instead. I think it would work then.